spacemesh/v2alpha1/block.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+import "spacemesh/v2alpha1/tx.proto";
+
+package spacemesh.v2alpha1;
+
+message Block {
+  oneof versioned {
+    BlockV1 v1 = 1;
+  }
+}
+
+message BlockV1 {
+  bytes id = 1; // block hash
+  repeated Transaction transactions = 2; // block transactions, in canonical order
+}
```

Comment:
i think this is also not so useful, would be better to report block hash and provide List/Stream apis for rewards and transactions